src/components/EolPage.js

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from 'react';
+import { format, parseISO } from 'date-fns';
+import { ja, ko } from 'date-fns/locale';
+
+const AGENT_RELEASE_NOTES_URL =
```

Comment:
do we want to use that JSON as source of truth or release notes currently in file system via a static query? currently that JSON is updated daily ~7p pacific.

Reply:
It will coincide with what we're supporting for the product so I thought it'd be ok, but we can refactor this to use the file system

Reply:
not 100% sure if one is better than the other. just wanted to think through it.

if agent team PRs repo to add release note and site builds, i imagine it's reasonable to expect that EoL page is in sync with release notes currently on site. 

in product agent recommendations gets updated data a bit later when all is live on site (in a pinch we can trigger workflow manually to update agent recs in product). until we have a better way to programmatically trigger the release note json workflow when the build is live.